### DISCUSSION TITLE
can't open lokal playbooks

### DISCUSSION THREAD

**Discussion by michaelklnsag** [semaphoreui/semaphore#1800] [General]
I can't open local playbooks, i even understands how to do it, documentation doesn't help me anyway? 

do i need a plain folder, with access rights, 
do i need a local git repo, 
do i need to link a folder, 
do i need to link a yaml file containing my playbooks?

**Comment by antiphasis** [comment]
- create an empty key in the key store
- for local playbooks enter the full path in the repository configuration and choose the empty key for the Access Key

Accessing a playbook in the Task Templates: You have to enter the name of the .yml file
